### PR TITLE
feat: [SRM-10035]: Changes Table changes for SLO Details

### PR DESCRIPTION
**Pull request by shashidhargg**
##### Summary:

Changes Table support in SLO Details

##### Jira Links:

https://harness.atlassian.net/browse/SRM-10035

##### Screenshots:

Screenshot 2022-04-19 at 8 14 23 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by deepesh-ui** [comment]
the space between ticker arrow and percentage is  more than expected 
image

**Comment by shashidhargg** [comment]
@deepesh-ui it's an existing component. You can see it in the Monitored Service details. I am just using it here.

File: src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/ChangesTable.tsx [hunk 1]
```diff
@@ -151,95 +154,91 @@ export default function ChangesTable({
...
-              pageIndex,
-              pageCount: totalPages,
-              itemCount: totalItems,
-              gotoPage: setPage
-            }}
-          />
-        </Card>
-      )
-    }
+  const Wrapper: React.FC = ({ children }) => {
```
Comment (praneshg239) [inline src/modules/85-cv/pages/monitored-service/components/ServiceHealth/components/ChangesAndServiceDependency/components/ChangesTable/ChangesTable.tsx:157]: Can we extract this to separate component?

**Review by praneshg239 — APPROVED** [review]

**Review by praneshg239 — CHANGES_REQUESTED** [review]
Please check the comment

**Review by deepesh-ui — DISMISSED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**